src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,16 @@
-const fizzBuzz = (number) => {};
+const fizzBuzz = (number) => {
+    const isDivisibleBy3 = number % 3 === 0;
+    const isDivisibleBy5 = number % 5 === 0;
 
-module.exports = fizzBuzz;
+    if (isDivisibleBy3 && isDivisibleBy5) {
+        return "FizzBuzz";
+    }
+    if (isDivisibleBy5) {
+        return "Buzz";
+    }
+    if (isDivisibleBy3) {
+        return "Fizz";
+    }
+        return number;
+  };
+  module.exports = fizzBuzz;
```

Comment:
Since most of your code is indented to 2 spaces, Im going to assumer thats what you meant to use so you should update this function:
```
const fizzBuzz = (number) => {
  const isDivisibleBy3 = number % 3 === 0;
  const isDivisibleBy5 = number % 5 === 0;

  if (isDivisibleBy3 && isDivisibleBy5) {
    return "FizzBuzz";
  }
  if (isDivisibleBy5) {
    return "Buzz";
  }
  if (isDivisibleBy3) {
    return "Fizz";
  }
  return number;
};

module.exports = fizzBuzz;
```
There should always be a clear line before the module.exports statement at the end of the file, and the closing bracket of the function should be the same level of indentation as the opening of the function, so in this case at the far left